CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -164,13 +168,13 @@ public void CanRemindLateDevelopersEveryHalfHourUntilOneThirty(int hour, int min
         [Test]
         public void CanRemindLearnTechChannelAtOneThirty()
         {
-            var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
-            
             var clock = new ClockStub(
```

Comment:
Maybe the ClockStub could go to a setup method along with getLateDeveloper to reduce duplication?

Reply:
Ignore this, Yusuf commented the same

Reply:
The clock changes in each test. How can this be in setup?

Reply:
No, it cannot go to the `SetUp` method because the clock changes.